Routes/VendorAppApi.js

```diff
@@ -76,7 +77,7 @@ router.get('/', function(req, res) {
             }
         ]
     };
-    
+
     res.json(vendorAppData);
 });
 
```

Comment:
Nit: can remove some of these spaces before 'module.exports = router;', but code looks clean overall!